### PR TITLE
Feature/mob 8 accept order

### PR DESCRIPTION
**Pull request by albertusfavian**
# Description
Accept Order and Notify User

# Ticket
https://berkatjayagroup.atlassian.net/browse/MOB-8?atlOrigin=eyJpIjoiM2RiNjNjOTMxYzVjNGQwNWEwZDU5ZjU2MjdkNGU5NDgiLCJwIjoiaiJ9

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

File: Moku.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -277,6 +287,7 @@
 				832AC1E42715986A00BD92E2 /* SwiftUIX in Frameworks */,
 				837370C527144BB1003656FC /* FirebaseAuth in Frameworks */,
 				837370C727144BB1003656FC /* FirebaseFirestore in Frameworks */,
+				8ED736342742CA010043B882 /* HalfASheet in Frameworks */,
```
Comment (naufaldirfq) [inline Moku.xcodeproj/project.pbxproj]: bert HalfASheetnya diapus sekalian aja dari SPM

Comment (albertusfavian) [reply Moku.xcodeproj/project.pbxproj]: yaudah mangga a

File: Moku/MokuApp.swift [hunk 2]
```diff
@@ -17,22 +18,29 @@ struct MokuApp: App {
...
     var body: some Scene {
         WindowGroup {
-            if let user = session.user {
-                switch user {
-                case .bengkel(_):
-                    BengkelView()
-                case let .customer(customer):
-                    CustomerView(for: customer)
-                }
+//             contentView()
```
Comment (naufaldirfq) [inline Moku/MokuApp.swift]: balikin ke contentView lagi

File: Moku/MokuApp.swift [hunk 3]
```diff
@@ -51,7 +59,7 @@ class AppDelegate: NSObject, UIApplicationDelegate {
 
         NotificationService.register(application: application)
 
-//        BengkelRepository.shared.add(bengkel: .preview)
+        //        BengkelRepository.shared.add(bengkel: .preview)
```
Comment (naufaldirfq) [inline Moku/MokuApp.swift]: apus aja kalo gak dipake

File: Moku/MokuApp.swift [hunk 4]
```diff
@@ -17,22 +18,29 @@ struct MokuApp: App {
...
+        if let user = session.user {
+            switch user {
+            case .bengkel(_):
+                BengkelView()
+            case let .customer(customer):
+                CustomerView(for: customer)
+            }
+        } else {
+            if appState.hasOnboarded {
+//                AssignMechanicsView().environmentObject(sheetManager)
```
Comment (naufaldirfq) [inline Moku/MokuApp.swift]: apus aja

File: Moku/Repositories/BengkelRepository.swift [hunk 5]
```diff
@@ -92,4 +92,8 @@ final class BengkelRepository: ObservableObject {
                 ["reviews": FieldValue.arrayUnion([reviewDict])]
             )
     }
+
+    func getMechanics(bengkelId: String) {
+
+    }
```
Comment (naufaldirfq) [inline Moku/Repositories/BengkelRepository.swift:98]: apus aja kalo gak dipake

File: Moku.xcodeproj/project.pbxproj (hunk 1 above)
Comment (naufaldirfq) [reply Moku.xcodeproj/project.pbxproj]: elu yg apus ya bertt

**Review by naufaldirfq — APPROVED** [review]

**Pull request merged**